src/parse/parser.rs

```diff
@@ -385,14 +389,7 @@ impl<'help, 'app> Parser<'help, 'app> {
                     ParseResult::Opt(_) | ParseResult::Pos(_)
                         if !self.is_set(AS::SubcommandPrecedenceOverArg) => {}
                     _ => {
-                        let sc_name = self.possible_subcommand(&arg_os);
-                        debug!(
-                            "Parser::get_matches_with: possible_sc={:?}, sc={:?}",
-                            sc_name.is_some(),
-                            sc_name
-                        );
```

Comment:
You removed this debug. Please don't remove debugs.

Reply:
Yep.